projects/canopy/src/styles/mixins.scss

```diff
@@ -160,5 +160,23 @@ $breakpoints: (
     background-color: transparent !important;
     border-color: var(--#{$variant}-color) !important;
     color: var(--#{$variant}-color) !important;
+
+    &:active,
+    &:active:hover {
+      background-color: var(--#{$variant}-link-active-hover-color) !important;
+      color: var(--color-white) !important;
+    }
+
+    @if ($variant == 'generic') {
+      &:hover {
```

Comment:
Can I just say that I don't like the hover state for the generic button? 🤣  why doesn't it follow the same pattern as the other ones?

Reply:
totally understandable, I don't like it either :)). The reason for this is that we already have a `--generic-link-hover-color` defined and it is not the color we need for the generic hover state, from the designs. I could not change the existing `--generic-link-hover-color` as that one is used for the link in the generic variant. So this was my solution for using `--color-taupe-grey` as background for the button in the generic variant, without breaking the links. Hope that makes sense :).

Reply:
I wonder if it's worth just running this by design again, as there may have been a mistake in the designs? They may change the designs if it's more consistent with the other ones.

Reply:
Will run this by design again and see what we should  do. :)

Reply:
Oh I think the issue is that the border color is supposed to be the same `--color-taupe-grey` on hover as the background.

Reply:
@owensgit Had a chat with Michelle this morning and she thinks this is ok for now. This will be looked at again when the component that combines "Variants", "Alerts" and "Details" is going to be designed and built.
@elenagarrone yep, will do that now.

Michelle said that if we need to discuss this further she will be happy to jump on a call with us :).